aws/errors.go

```diff
@@ -0,0 +1,60 @@
+package aws
+
+import (
+	"context"
+	"path"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+)
+
+// isNotFoundError:: function which returns an ErrorPredicate for AWS API calls
+func isNotFoundError(notFoundErrors []string) plugin.ErrorPredicateWithContext {
+	return func(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData, err error) bool {
+		awsConfig := GetConfig(d.Connection)
+
+		// If the get or list hydrate functions have a overriding IgnoreConfig
+		// defined using isNotFoundError function then
+		// It should also check for errors in "ignored_error_codes"
+		allErrors := append(notFoundErrors, awsConfig.IgnoredErrorCodes...)
+		if awsErr, ok := err.(awserr.Error); ok {
+			plugin.Logger(ctx).Info("isNotFoundError", "AWS Error CODE", awsErr.Code())
```

Comment:
Do we still want to include these as part of the release? Or do we need to modify the error message/change the level?